src/process/tags.md

```diff
@@ -0,0 +1,56 @@
+# Process tags
+
+Description of tags, used for projects.
+
+## Status
+- refine `#c377e0` - task already have resolving code, but code needs some fixes.
+- take it `#0052cc` - any teammate can take this task and start working with it.
+- discuss `#fbca04` - task needs to finish discussion before start working.
+- needs info `#c2e0c6` - task needs additional requirements to start working.
+- blocked `#f8c700`
+- blocker `#cc0701`
+
+## Niceness
+- easy `#fef2c0` - new teammates can start with such tasks to become familiar with project.
```

Comment:
new teammates **may** start with such **issue** to become familiar with **a** project

Reply:
@artemiy312 , emm, either Architect or Manager decides who can take task. That's why new teammates can, but not may.

But i can fix it, if you insist